homeassistant/components/picnic/const.py

```diff
@@ -0,0 +1,102 @@
+"""Constants for the Picnic integration."""
+from homeassistant.const import CURRENCY_EURO, DEVICE_CLASS_TIMESTAMP
+
+DOMAIN = "picnic"
+
+CONF_API = "api"
+CONF_COORDINATOR = "coordinator"
+CONF_COUNTRY_CODE = "country_code"
+
+COUNTRY_CODES = ["NL", "DE", "BE"]
+ATTRIBUTION = "Data provided by Picnic"
+ADDRESS = "address"
+
+SENSOR_COMPLETED_DELIVERIES = "completed_deliveries"
+SENSOR_TOTAL_ORDERS = "total_orders"
```

Comment:
Some of these sensors are… not very useful?

Reply:
I've removed some not really useful sensors and made a bunch of others disabled by default (as suggested below).